notebooks/HST_WFC3_w_ExoMAST.ipynb

```diff
@@ -20,9 +20,20 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
```

Comment:
Hey @nespinoza I am looking at the changes and it looks like you are submitting notebooks that have exectuted cells. Not that there is anything wrong with that, but when you are dealing with executed notebooks, they take up more space and also makes changes difficult to track.

Reply:
Hey @mfixstsci! I agree --- however, those come directly from the merge I did with Natasha's repo (i.e., this commit https://github.com/nespinoza/PandExo/commit/c16e3f35a38dd2adeafdb2798f76e5a470e36838), so that's either Natasha or Kevin leaving those notebooks with executed cells. I suggest we keep it that way (again, because that will make it less of a headache for Natasha to review this) and, if we want, submit another PR editing those cells.